{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2016-6828",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "linux",
            "product": {
              "product_data": [
                {
                  "product_name": "linux_kernel",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "4.7.4"
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-416"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=bb1fceca22492109be12640d49f5ea5a544c6bb4"
        },
        {
          "url": "http://www.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.7.5"
        },
        {
          "url": "http://www.openwall.com/lists/oss-security/2016/08/15/1"
        },
        {
          "url": "http://www.securityfocus.com/bid/92452"
        },
        {
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1367091"
        },
        {
          "url": "https://github.com/torvalds/linux/commit/bb1fceca22492109be12640d49f5ea5a544c6bb4"
        },
        {
          "url": "https://marcograss.github.io/security/linux/2016/08/18/cve-2016-6828-linux-kernel-tcp-uaf.html"
        },
        {
          "url": "https://source.android.com/security/bulletin/2016-11-01.html"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The tcp_check_send_head function in include/net/tcp.h in the Linux kernel before 4.7.5 does not properly maintain certain SACK state after a failed data copy, which allows local users to cause a denial of service (tcp_xmit_retransmit_queue use-after-free and system crash) via a crafted SACK option."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe": [
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/o:linux:linux_kernel",
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "4.7.4"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "version": "3.0",
        "vectorString": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
        "attackVector": "LOCAL",
        "attackComplexity": "LOW",
        "privilegesRequired": "LOW",
        "userInteraction": "NONE",
        "scope": "UNCHANGED",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "HIGH",
        "baseScore": 5.5,
        "baseSeverity": "MEDIUM"
      },
      "exploitabilityScore": 1.8,
      "impactScore": 3.6
    },
    "baseMetricV2": {
      "cvssV2": {
        "version": "2.0",
        "vectorString": "(AV:L/AC:L/Au:N/C:N/I:N/A:C)",
        "accessVector": "LOCAL",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "COMPLETE",
        "baseScore": 4.9
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 3.9,
      "impactScore": 6.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2016-10-16T21:59Z",
  "lastModifiedDate": "2017-01-03T03:00Z"
}